packages/calypso-analytics/src/utils/current-user.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Internal dependencies
+ */
+import hashPii from './hash-pii';
+
+/**
+ * External dependencies
+ */
+import { UserData } from '../../../../client/lib/user/user';
```

Comment:
Will this work? Do we want it to work? Generally, I wouldn't expect packages to import anything from the client app.

Reply:
yea that was a bit crazy I took out that reference.